### PR TITLE
Preacher and spawn fixes

### PR DESCRIPTION
**Pull request by roboticxlink**
Fixes so that preacher has a place to spawn again (yay) 

Also changes to Preacher that puts it more in line to TG chaplain. Null Rod, Bible, isHoly. Etc

Fixes a few spawn bugs on sheriff, and settler. Also reworks Survival Box item to contain a set of more native items for survival.

### CODE REVIEW

File: code/modules/jobs/job_types/den.dm [hunk 1]
```diff
@@ -77,6 +77,125 @@ Settler
...
+			B.name = "The Book of Lorgar"
+		if("imperium")
+			B.name = "Uplifting Primer"
+		if("toolboxia")
+			B.name = "Toolbox Manifesto"
+		if("homosexuality")
+			B.name = "Guys Gone Wild"
+		if("lol", "wtf", "gay", "penis", "ass", "poo", "badmin", "shitmin", "deadmin", "cock", "cocks", "meme", "memes")
+			B.name = pick("Woodys Got Wood: The Aftermath", "War of the Cocks", "Sweet Bro and Hella Jef: Expanded Edition")
+			H.adjustBrainLoss(100) // starts off retarded as fuck
```
Comment (LetterN) [inline code/modules/jobs/job_types/den.dm]: kek

Comment (roboticxlink) [reply code/modules/jobs/job_types/den.dm]: Pffft. That's all apparently in TG chaplain.

**Comment by LetterN** [comment]
conflict thingy

**Comment by roboticxlink** [comment]
@lettern where's the conflict?

**Comment by LetterN** [comment]
image

**Comment by roboticxlink** [comment]
All my map change did was add a spawn for the preacher *screams* what other map changes were done? Because yeah. Idk what they changed.

**Comment by roboticxlink** [comment]
@LetterN I FIXED IT AAAHHH

**Pull request merged**